.github/workflows/ios-build-test.yml

```diff
@@ -15,6 +15,16 @@ jobs:
       - name: checkout repository
         uses: actions/checkout@v3
 
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          cache: 'npm'
```

Comment:
Btw, from my experience sometimes caching is slower than not caching npm packages... Consider simplifying it and not caching.

Reply:
OK, we can skip for now and measure; I'll do 2 runs to see which one is faster and if there's no difference, will remove. There's relatively few packages now anyway.

Reply:
Right. With caching it's 4s (cache unpack, npm ci, cache pack) vs 3s or 2s. So I'll remove it. The time is negligible anyway.